README.md

```diff
@@ -1,33 +1,33 @@
-# Introduction
+![Tests](https://img.shields.io/github/workflow/status/OpenMined/PyDP/Tests)
+![Version](https://img.shields.io/github/v/tag/OpenMined/PyDP?color=green&label=pypi)
+![License](https://img.shields.io/github/license/OpenMined/PyDP)
 
-PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information.
 
-PyDP is part of the OpenMined community, come join the movement on [Slack](http://slack.openmined.org/).
+# PyDP
 
-# Instructions
+In today's data-driven world, where data analytics is used by the Researchers or Data Scientists to create better models or innovative solutions for a better future! They often tend to handle sensitive or personal data, which brings in some privacy concerns. For example, sometimes, AI models can memorize details about the data they've trained on and could 'leak' these details later on. Differential privacy is a framework (using math) for measuring this leakage and reducing the possibility of it happening. 
 
-If you'd like to contribute to this project please read these [guidelines](https://github.com/OpenMined/PyDP/blob/master/contributing.md).
+This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, helping us achieve better privacy.
 
+**Things to remember about PyDP :**
+- :rocket: Features Differentially Private Algorithmic functions as part of the 0.1.1 dev release, added to carrots demo such as BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median , etc.  
+- All the computation methods mentioned above, use Laplace noise only. (Other noise mechanisms will be added soon... :smiley:)
+- :fire: Currently supports Linux only. (OSX, Windows coming real soon... :smiley:)
```

Comment:
We have also added support for OS X .

Reply:
okey so we now support both osx and linux...